omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -93,7 +93,7 @@ for i in /opt/{chef,$path}*/version-manifest.txt \
     fi
 done
 
-for i in `find /var/log/${path}* -type f -mmin -"$modified_within_last_x_minutes"`; do
+for i in `find -L /var/log/${path}* -type f -mmin -"$modified_within_last_x_minutes"`; do
```

Comment:
[unrelated] Hmm, looking at this, I wonder if the `*` behind `${path}` really is needed. But I suppose it doesn't hurt either.